src/index.html

```diff
@@ -15,8 +16,334 @@
+          <div class="detail__imgs">
+            <img
+              src="./images/detail0201.png"
+              alt="#"
+              class="detail__img detail__img--wide"
+            />
+            <img
+              src="./images/detail0202.png"
+              alt="#"
+              class="detail__img detail__img--square"
+            />
+          </div>
+          <h2 class="detail__title">Range & Integrated lights</h2>
+          <p class="detail__description">
+            The removable battery has up to 70km battery autonomy and weighs
+            only 2.4 kg. Lights integrated into the frame give you always-on
+            visibility day and night.
+          </p>
+        </div>
+        <div class="detail">
+          <div class="detail__imgs">
+            <img
+              src="./images/detail0301.png"
+              alt="#"
+              class="detail__img detail__img--square"
+            />
+            <img
+              src="./images/detail0302.png"
+              alt="#"
+              class="detail__img detail__img--wide"
+            />
+          </div>
+          <h2 class="detail__title">Hydraulic disc brakes & Lightweight</h2>
+          <p class="detail__description">
+            Brakes with total stopping power the second you make contact. The
+            removable battery has up to 70km battery autonomy and weighs only
+            2.4 kg. Lights integrated into the frame give you always-on
+            visibility day and night.
+          </p>
+        </div>
+        <button class="detail__btn">Explore</button>
+      </section>
+      <!-- #endregion -->
+      <!-- #region contact -->
+      <section
+        class="contact"
+        id="contact"
+      >
+        <h1 class="contact__section section__title">Contact us</h1>
+        <form
+          onsubmit="this.reset(); return false;"
+          action="submit"
+          class="contact__form"
+        >
+          <input
+            placeholder="Name"
+            type="text"
+            class="contact__input contact__form__name"
+            required
+          />
+          <input
+            placeholder="Email"
+            type="email"
+            class="contact__input__email"
+            required
+          />
+          <textarea
+            placeholder="Message"
+            type="text"
+            class="contact__input contact__form__msg"
+          ></textarea>
+          <button
+            type="submit"
+            class="contact__form__submit"
+          >
+            Send
+          </button>
+        </form>
+        <div class="contact__info">
+          <div class="contact__phones">
+            <p class="contact__title">Phone</p>
+            <p class="contact__content">
+              <a href="tel:+12345555555">+1 234 555-55-55</a>
+            </p>
+          </div>
+          <div class="contact__emails">
+            <p class="contact__title">Email</p>
+            <p class="contact__content">hello@miami.com</p>
```

Comment:
it's missing the link attribute
<a href="hello@miami.com">
<p class="contact__content">hello@miami.com</p>
</a>